First_last_position_sortedarray.py

```diff
@@ -0,0 +1,21 @@
+class Solution:
+    def binarysearch(self, nums, target, left):
```

Comment:
There is no such method name in leetcode question. How did you manage to run this?